src/App.js

```diff
@@ -6,15 +6,17 @@ import Navbar from './components/Navbar';
 import './App.css';
 
 const App = () => (
-  <Router>
-    <Navbar />
-    <div>
-      <Routes>
-        <Route path="/books" element={<Books />} exact />
-        <Route path="/categories" element={<Categories />} exact />
-      </Routes>
-    </div>
-  </Router>
+  <div className="panel-bg">
+    <Router>
+      <Navbar />
+      <div>
+        <Routes>
+          <Route path="/books" element={<Books />} exact />
+          <Route path="/categories" element={<Categories />} exact />
+        </Routes>
```

Comment:
- [ ] I think there is a bug with the books component, when the user moves from books to categories and back to books the books are being dispatched again and being added to the store. We can fix this by making sure before fetching the books again that our store is empty.